lunes_cms/static/js/color_unread_feedback.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Function to highlight unread feedback with bold letters
+*/
+
+
+function markUnreadFeedback () {
+    const readByEntries = document.querySelectorAll(".field-read_by");
+    if (readByEntries.length > 0) {
+        
+        readByEntries.forEach((entry)=>{
+            if (entry.textContent == "-") {
+                entry.parentNode.style.backgroundColor = "orange";
+                entry.parentNode.style.fontWeight = "bold";
+            }
```

Comment:
```suggestion
            } else {
            entry.parentNode.style.fontWeight = "normal";
           }
```
If I'm not mistaken I think we have to do it this way and can't do it via HTML as we the font-weight is inherited from the theme - but I don't feel 100% sure about it